### ISSUE TITLE
Facebook access_token_method

### ISSUE THREAD

**Issue by marfire** [pennersr/django-allauth#704]
In researching #703 I noticed that the [Facebook documentation](https://developers.facebook.com/docs/facebook-login/manually-build-a-login-flow/v2.1) uses a `GET` to the `https://graph.facebook.com/oauth/access_token` endpoint. I can't find anything in the (sketchy) documentation that mentions a `POST` to that url.

`Allauth` is currently doing the default `POST`. That works, obviously, but overriding with `access_token_method = 'GET'` would seem to be more conformant?

**Comment by derek-adair** [comment]
Where can this be done? seems like some very low hanging fruit to clean up.

Worth noting - if this hasn't been an issue in 9 years it's maybe not a big deal.

**Comment by pennersr** [comment]
@derek-adair Definitely not a big deal, though theoretically FB might become more strict in a future API version and break this. It's just a matter of setting the default:

```
class OAuth2Adapter(object):
    access_token_method = "POST"
```

to:

```
class FacebookOAuth2Adapter(OAuth2Adapter):
    access_token_method = "GET"
```

... and then confirm FB login still works.

**Comment by pennersr** [comment]
Closed via 3d685a26

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/704
